src/Components/HomePage.js

```diff
@@ -128,6 +129,43 @@ const Theory = props => {
 };
 
 export default HomePage = props => {
+  const { navigation } = props;
+
+  const feedback = () => {
+    Alert.alert(
+      I18n.t('feedback.contact.title'),
+      I18n.t('feedback.contact.content'),
+      [
+        { text: I18n.t('feedback.contact.cancel'), style: 'cancel' },
+        { text: I18n.t('feedback.contact.cta'), onPress: sendEmailAsync },
+      ],
+      { cancelable: true },
+    );
+  };
+
+  const sendEmailAsync = () => {
+    let result = MailComposer.composeAsync({
+      recipients: ['ultimate.discin@gmail.com'],
+      subject: 'Feedback Disc In',
+      body: '',
+    });
+
+    alert(result.status);
+  };
+
+  const displayFeedbackButton = () => {
```

Comment:
Is it possible to extract the code related to feedback in another component si that the HomePage one does not include everything (it could only display the button) ?